### DISCUSSION TITLE
How do I build a page to get the link card I want?

### DISCUSSION THREAD

**Discussion by jghiloni** [bluesky-social/atproto#2595] [How to do it?]
I'm working on a service that will show US National Weather Service alerts in a feed, and I will have a custom site to show the affected area and the necessary info from the alert. What does the app use to build the link card? I'm curious what I need to put into the page to have an ideal link card generated. Specifically, the site will be dynamically generated, so if there's meta information that needs to be included to get that data, I'd love to know about it. Links to the source are perfectly fine for me. Thanks!

**Comment by bnewbold** [comment]
I think we pretty much use Open Graph metadata by default, with some other HTML meta tags and signals used if that doesn't work; eg Twitter-style metadata as a fallback. Open Graph is the closest thing to an interoperable standard for this specific use-case (despite starting as a Facebook thing), and I would recommend using that.

https://ogp.me/

https://en.wikipedia.org/wiki/Facebook_Platform#Open_Graph_protocol
